docs: Update Studio/HD documentation with latest details (#20209)
unoplatform/uno: doc/articles/studio/Hot Design/hot-design-properties.md

---
uid: Uno.HotDesign.Properties
---

# Properties

## General

The Properties window allows you to view and edit the attributes of any element selected on the canvas or in the Elements window. As soon as an element is selected, its properties are loaded into the Properties window, allowing you to make changes based on the property's type and expected value.

### Search

You can quickly find a specific property by using the search box located below the **Style** section. To open the search input, click on the magnifying glass icon. As you type the property's name, the results will be filtered in real time. To clear your search, click the **X** button inside the search box.

<img src="Assets/properties-search.gif" height="600" alt="GIF showing how to Search for properties on the Properties window" />

### Smart and All tabs

At the top of the Properties window, you'll see two tabs: **Smart** and **All**.

- **Smart** displays the most commonly adjusted properties for the selected element, providing quicker access for typical use cases.
- **All** shows the complete list of properties available for that element.

> Just click the tab you want to switch views.

Reviewer: ```suggestion
Just click the tab you want to switch between views.
```

<img src="Assets/properties-smart-tab.gif" height="600" alt="GIF showing how to switch between tabs on the Properties window" />

### Expandable Sections

To make things easier to navigate, properties are grouped into sections such as *Identity*, *Common*, *Templates*, *Interactions*, and others - depending on the selected element. You can expand or collapse these groups by clicking the arrow next to the group title. This helps reduce visual clutter and lets you focus only on the property groups you're working with.

<img src="Assets/properties-expandable-sections.gif" height="600" alt="GIF showing how to expand and collapse sections on the Properties window" />

### Assistive Text

> If you enter something wrong in a property, a message will show up just below it. A yellow message means a warning, and a red one means an error. This helps you understand what went wrong so you can fix it quickly. <!-- TODO: Add screenshot?  -->

Reviewer: ```suggestion
If you enter an incorrect value in a property, a message will show up just below it. A yellow message means a warning, and a red one means an error. This helps you understand what went wrong so you can fix it quickly. <!-- TODO: Add screenshot?  -->
```

## Applying Styles

> You can apply predefined styles to your elements. To do this, go to the Style section at the top of the Properties window. There you'll see a field called "Implicit Style." Click it to view the available styles for the selected element, or type in the name to filter the list. Once you click a style, it will be applied to your element. To remove it, click the more options icon next to the field and choose "Reset."

Reviewer: ```suggestion
You can apply predefined styles to your elements. To do this, go to the Style section at the top of the Properties window. There you'll see a field called "Implicit Style". Click it to view the available styles for the selected element, or type in the name to filter the list. Once you click a style, it will be applied to your element. To remove it, click the "More Options" icon next to the field and choose "Reset".
```
